### PR TITLE
gives zombies a big bad message

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

zombies now get a message to not be pussies and act like a zombie when they first become one

## Why It's Good For The Game

FUCK

## Changelog
:cl:
add: a new line in the you are a zombie message has been added
/:cl:

### CODE REVIEW

**Comment by ishitbyabullet** [comment]
**WELL BOYS, WE DID IT**
image
**FRIENDLY ZOMBIES ARE NO MORE**

**Comment by Akrilla** [comment]
thank god. finally zombies won't fuckin run to medbay to get cloned

**Comment by Fikou** [comment]
prepare for my civil rights beam

**Comment by ATH1909** [comment]
Maybe you should specify that they shouldn't help non-zombie creatures _with brains_?

**Comment by Fikou** [comment]
> Maybe you should specify that they shouldn't help non-zombie creatures _with brains_?

NO!!

File: code/modules/zombie/organs.dm [hunk 1]
```diff
@@ -90,7 +90,7 @@
 	playsound(owner.loc, 'sound/hallucinations/far_noise.ogg', 50, 1)
 	owner.do_jitter_animation(living_transformation_time)
 	owner.Stun(living_transformation_time)
-	to_chat(owner, "<span class='alertalien'>You are now a zombie!</span>")
+	to_chat(owner, "<span class='alertalien'>You are now a zombie! Do not seek to be cured, do not help any non-zombies in any way, and do not harm your zombie brethren. You are a creature of hunger and violence.</span>")
```
Comment (MrDoomBringer) [inline code/modules/zombie/organs.dm]:
```suggestion
	to_chat(owner, "<span class='alertalien'>You are now a zombie! Do not seek to be cured, do not help any non-zombies in any way, do not harm your zombie brethren, and kill as many non-zombies as possible. You are a creature of hunger and violence.</span>")
```

**Comment by Tlaltecuhtli** [comment]
test merge this on sybil (baby server)

File: code/modules/zombie/organs.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/zombie/organs.dm]: i changed it a bit

**Comment by dr-bee** [comment]
there goes my patented "help me pleaCHOMP" strat that makes eating doctors a breeze.

**Comment by SpaceManiac** [comment]
Is there admin support for this?

**Comment by Fikou** [comment]
> Is there admin support for this?

me no understand

**Review by SpaceManiac — APPROVED** [review]
w/e, we can revert it or make it a config if people complain

**Pull request merged**